packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/react",
-  "version": "0.12.7",
+  "version": "0.13.0",
```

Comment:
Confusingly, npm will treat this as effectively a new major version, which means `npm update @gadgetinc/react` won't ever move the version for existing apps. Since this isn't a breaking change, can we make it a patch version instead? I don't feel super strongly. This is also why that `versions` test is failing

Reply:
There is a breaking change to `GadgetClientContext` where before it simply returned the `api` client, and now it is called `GadgetConfigurationContext` and returns `{ api, auth }`

I think that warrants an major/minor version bump?

Reply:
I'm cool with that, but I think that context is best considered private to the implementation such that the provider should really be the thing that accepts all input you know